releasing.rst

```diff
@@ -151,10 +166,8 @@ We do this in two ways for each release.
 All release announcements should be posted
 on the `forum <https://forum.nengo.ai/c/general/announcements>`_
 and on the `ABR website <http://appliedbrainresearch.com/>`_.
-Links to the announcements should be posted
-on `Twitter <https://twitter.com/abr_inc>`_.
 
-For major release
+For a major release
```

Comment:
Yeah, we haven't been using twitter very consistently and I don't think we're likely to start. Hopefully people realize these are just suggestions of where to promote releases.